home/tests/test_api.py

```diff
@@ -858,8 +859,10 @@ def create_content_page(
         list_items: list[str] | None = None,
         next_prompt: str | None = None,
         footer: str | None = None,
+        whatsapp_template_example_values: list[ExVal] | None = None,
+        whatsapp_template_image: int | None = None,
         whatsapp_template_category: str | None = None,
-        # whatsapp_template_submission_name: str | None = None,
+        whatsapp_template_submission_name: str | None = None,
         whatsapp_template_slug: str | None = None,
```

Comment:
Do these fields still exist on `ContentPage`?

Reply:
Yes, we use them to create the template inside `create_content_page`